crates/sncast/Cargo.toml

```diff
@@ -42,19 +41,19 @@ cairo-vm.workspace = true
 blockifier.workspace = true
 semver.workspace = true
 console.workspace = true
-sha3 = "0.10.8"
-base16ct = { version = "0.2.0", features = ["alloc"] }
+sha3.workspace = true
+base16ct.workspace = true
 
 [dev-dependencies]
 ctor.workspace = true
 snapbox.workspace = true
 indoc.workspace = true
 project-root.workspace = true
-reqwest = "0.11.18"
-tempfile = "3.10.1"
+reqwest.workspace = true
```

Comment:
It is both in dev-deps and in dependencies, is it on purpose? @THenry14

Reply:
I don't think this is necessary, removed from dev deps